src/AccessibilityInsights.SharedUx/Telemetry/ITelemetrySink.cs

```diff
@@ -7,10 +7,15 @@ namespace AccessibilityInsights.SharedUx.Telemetry
 {
     internal interface ITelemetrySink
     {
+        /// <summary>
+        /// We allow group policy to disable telemetry. This takes precedence over user opt-in
+        /// </summary>
+        bool DoesGroupPolicyAllowTelemetry { get; }
+
         /// <summary>
         /// Whether or not telemetry toggle button is enabled in the settings.
         /// </summary>
-        bool IsTelemetryAllowed { get; set; }
+        bool HasUserOptedIntoTelemetry { get; set; }
```

Comment:
This is just a rename to better distinguish which level of enabling we are setting

Reply:
Done